test/integration/base.py

```diff
@@ -260,6 +260,8 @@ def _assert_namedtuple(self, a, b):
                 self.assertRegex(v_b, v_a)
             elif isinstance(v_a, tuple) and isinstance(v_b, tuple):
                 self._assert_namedtuple(v_a, v_b)
+            elif isinstance(v_a, list) and isinstance(v_b, list):
+                self._assert_namedtuple(v_a, v_b)
```

Comment:
@seanmacavaney This might be something for you to check before merging.
I figured that lists are otherwise only checked for equality, not with respect to regexes like tuples.
The tuple comparison function works for lists too, however, so I just used that.

Reply:
seems fine to me